app/views/layouts/angular/_paging_div_buttons.html.haml

```diff
@@ -1 +1 @@
-%div#angular_paging_div_buttons
+#angular_paging_div_buttons{:align => "right"}
```

Comment:
`{:align => "right"}` is not needed for any existing users of this div.

It should not be needed for this form either.